x-pack/winlogbeat/winlogbeat.yml

```diff
@@ -38,6 +38,20 @@ winlogbeat.event_logs:
           id: sysmon
           file: ${path.home}/module/sysmon/config/winlogbeat-sysmon.js
 
+  - name: Windows PowerShell
```

Comment:
I think we ought to add an `event_id` filter to only pull the events we can process (and that are interesting).